clientd3d/preferences.c

```diff
@@ -0,0 +1,1718 @@
+#include "preferences.h"
+#include "client.h"
+#include "resource.h"
+
+#include <vector>
+
+// Preferences constants
```

Comment:
Make as many of these constants static as you can, so that they can't be referenced from outside the file.